<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetPaymentOperation</class>
 <widget class="QWidget" name="AssetPaymentOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="4" column="1" colspan="9">
    <widget class="QLineEdit" name="note"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="date_label">
     <property name="text">
      <string>Date/Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="number_label">
     <property name="text">
      <string>#</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QPushButton" name="commit_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Commit changes</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLineEdit" name="price_edit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="type_label">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="tax_label">
     <property name="text">
      <string>Tax</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLineEdit" name="number"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="symbol_label">
     <property name="text">
      <string>Asset</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="main_label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Asset Payment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="AccountCurrencyLabel" name="currency">
     <property name="text">
      <string>CUR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <spacer name="horizontal_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="tax_edit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDateEdit" name="ex_date_editor">
     <property name="specialValueText">
      <string>unknown</string>
     </property>
     <property name="minimumDate">
      <date>
       <year>1970</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="maximumTime">
      <time>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="AccountSelector" name="account_widget" native="true"/>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="AssetSelector" name="asset_widget" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="note_label">
     <property name="text">
      <string>Note</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="account_label">
     <property name="text">
      <string>Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="ex_date_label">
     <property name="text">
      <string>Ex-Date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QPushButton" name="revert_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Cancel changes</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="amount_label">
     <property name="text">
      <string>Dividend</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLineEdit" name="dividend_edit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QComboBox" name="type"/>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="timestamp_editor">
     <property name="displayFormat">
      <string>dd/MM/yyyy hh:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="price_label">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountSelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/reference_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AssetSelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/reference_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AccountCurrencyLabel</class>
   <extends>QLabel</extends>
   <header>jal/widgets/account_select.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
